src/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Project</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+    />
+  </head>
+  <body class="container">
+    <header>
+      <div class="logo text-white"></div>
+      <nav>
+        <ul class="menu">
+          <li class="menu-item active text-white item-counter">Tv Shows(59)</li>
+          <li class="menu-item text-white">Schedule</li>
+          <li class="menu-item text-white">Episodes</li>
+          <li class="menu-item text-white">People</li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
Please ensure to style the header accordingly in order for it to match the wireframe that was given to you. This would make your design similar to the original template.

Original design
<img width="401" alt="image" src="https://github.com/paulshaun-kdtk/Week5_API_Capstone/assets/84930355/ed53e234-1853-4f3a-bec7-ef9d7f042554">

Your design
<img width="960" alt="image" src="https://github.com/paulshaun-kdtk/Week5_API_Capstone/assets/84930355/36af6a30-4f06-4ed1-a300-f6a91e0d385d">